src/APIFetch.js

```diff
@@ -9,6 +9,16 @@ export const getSingleMovie =(id) => {
     })
 }
 
+export const getSingleMovieTrailer = (id) => {
+  return fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}/videos`)
```

Comment:
Nice work building this. I think we can refactor this to have both fetches in one for the single movie function, using a promise all?